src/types.js

```diff
@@ -128,6 +128,7 @@ export type QRCodeProps = {|
     state? : $Values<typeof QRCODE_STATE>,
     errorText? : string,
     onClose? : () => ZalgoPromise<void>,
+    onSubmitFeedback? : (reason : string) => void,
```

Comment:
Should we declare a map type?
```suggestion
    onSubmitFeedback? : (reason : Reasons[]) => void,
```

Reply:
Thank you for your suggestion David.
The parameter on the `onSubmitFeedback` is actually one single reason, on singular. That means we intent this to be a string.